### PR TITLE
Self merging gives -100 GBP

### PR DESCRIPTION
**Pull request by Cyberboss**
Requested by @optimumtact

### CODE REVIEW

**Comment by kevinz000** [comment]
NO!!!!

**Comment by kevinz000** [comment]
rip ever catching up to memetainers.

**Comment by Bawhoppen** [comment]
The rich get richer

**Comment by duncathan** [comment]
:+1: x 💯

File: tools/WebhookProcessor/github_webhook_processor.php [hunk 1]
```diff
@@ -505,11 +505,20 @@ function update_pr_balance($payload) {
 	if(!$trackPRBalance)
 		return;
 	$author = $payload['pull_request']['user']['login'];
+	$merger = $payload['pull_request']['merged_by']['login'];
 	$balances = pr_balances();
+	$friendliness = get_pr_code_friendliness($payload, $balances[$author]);
+	adjust_pr_balance($author, $friendliness, $balances);
+	if($author != $merger)
+		adjust_pr_balance($merger, 1, $balances);
```
Comment (duncathan) [inline tools/WebhookProcessor/github_webhook_processor.php]: add an else clause that removes 1 GBP for a selfmerge

**Review by duncathan — CHANGES_REQUESTED** [review]

**Comment by Qbopper** [comment]
only merge if closing prs gives -1 gbp

**Comment by Qbopper** [comment]
lmao, shit, or that

**Comment by Cyberboss** [comment]
@Qbopper 3 words
>I
>am
>goofball

**Comment by Qbopper** [comment]
what did he mean by this

**Review by duncathan — DISMISSED** [review]

**Comment by ShizCalev** [comment]
Mmm...

**Comment by Cyberboss** [comment]
Mmm on second thought, given that the GBP system is working so well, I don't want to alienate our contributors by making them not able to keep up with maintainers

**Comment by optimumtact** [comment]
This doesn't make any sense because the people who might do this are the kind of people who won't care about GBP anyway

**Pull request closed without merging**